packages/react-composites/package.json

```diff
@@ -136,12 +136,11 @@
     "husky": "^4.3.0",
     "if-env": "~1.0.4",
     "jest-fetch-mock": "^3.0.3",
-    "jest-junit": "~13.0.0",
-    "jest": "26.6.0",
+    "jest-junit": "~16.0.0",
+    "jest": "29.5.0",
     "json-stringify-safe": "^5.0.1",
     "nan": "^2.14.1",
     "node-forge": ">=1.0.0",
-    "playwright": "~1.22.2",
```

Comment:
Playwright direct dependency is removed because it's not required since version 1.34.0
Pleae see https://github.com/microsoft/playwright/releases/tag/v1.34.0 for more details